egs/aishell/asr/simple_v1/prepare.py

```diff
@@ -11,10 +12,8 @@
 from pathlib import Path
 
 import torch
-from aishell import prepare_aishell
-from lhotse import CutSet, Fbank, LilcomFilesWriter, combine
-from lhotse.augmentation import SoxEffectTransform
-
+from lhotse import CutSet, Fbank, LilcomHdf5Writer, combine
+from lhotse.recipes import prepare_aishell
```

Comment:
`prepare.py` of the LibriSpeech recipe from the latest master uses MUSAN
for data augmentation. Please see
https://github.com/k2-fsa/snowfall/blob/560737d98526f989a7b5a786125be7f58235fc80/egs/librispeech/asr/simple_v1/prepare.py#L13

Could you also update this file to use MUSAN?

Reply:
OK.